src/main/java/org/spongepowered/api/event/entity/ai/AITaskEvent.java

```diff
@@ -77,6 +77,14 @@
     }
 
     /**
+     * Fired when an {@link AITask} is updated every
+     */
+    @FactoryCodeCheck(errorMessage = "The target entity '%s' is not the owner of the goal '%s'!")
```

Comment:
Maybe it generates the code [here](https://github.com/SpongePowered/SpongeAPI/pull/1012/files#diff-1378043d9608b3c46ff7baa499da8e70R2343)